jira-workflows/goose/check-jira-tickets/recipe.yaml

```diff
@@ -0,0 +1,20 @@
+version: 0.1.0
+title: "Check if there is any Jira issue we could be interested in"
+description: "Check issues for {{ component }} in {{ project }}"
```

Comment:
Just thoughts on my end:

We need to experiment a bit what works best. Should we check explicitly for one component or should we just crawl all Jiras for such packaging tasks?

Reply:
> I started querying for all the open issues in the project, but it timeouts.

maybe we could try to filter issues containing "rebase to" or "update to" in their titles? But Valentin's suggestion as first step makes total sense. 

Also looking into the issues, trying to find some good test scenarios, I was thinking when issue is well defined like https://issues.redhat.com/browse/RHEL-97473, the description could be useful also in the rebase recipe (but this can be queried also in there directly).

> Also I thought we should probably let the "component owners" know that we are playing with their "issues"?

I think unless we do only read operations that's ok.